runhouse/servers/http/http_server.py

```diff
@@ -789,6 +790,13 @@ def get_status(request: Request, send_to_den: bool = False):
 
         return obj_store.status(send_to_den=send_to_den)
 
+    @staticmethod
+    @app.get("/logs")
+    @validate_cluster_access
+    def get_logs(request: Request, tail: int = DEFAULT_SURFACED_LOG_LENGTH):
```

Comment:
noting that rohin is currently working on a PR that handles logs very similarly, might be worth building on that (we can discuss further too): 
https://github.com/run-house/runhouse/pull/1152/files#diff-3e12105589672c203d48d9b023b833a328411f3a3c8d09f92a84ec63e59e003eR469